includes/Core/Modules/Modules.php

```diff
@@ -378,6 +378,23 @@ function( $setting, $setting_key ) use ( $old_values, $module_slug, &$changed_se
 			10,
 			2
 		);
+
+		add_filter(
+			'googlesitekit_features_request_data',
+			function( $body ) {
+				$active_modules    = $this->sort_modules( $this->get_active_modules() );
```

Comment:
Maybe it would be better to add the sorting right into the `get_active_modules()` method? I think it's a reasonable expectation that the return values of that method should be sorted already, similar to how the `get_available_modules()` return values are already sorted.

That makes the code easier to use as you don't have to remember to think about order everywhere, it just works out of the box.

You could change the `get_active_modules()` implementation to use a single `array_filter` call on the available modules instead of `array_merge` with two arrays, this way you wouldn't even need to sort again since the available modules are already sorted correctly.

Reply:
Beautiful - all _sorted_! 😃